packages/ckeditor5-dev-env/tests/release-tools/utils/generatechangelogfromcommits-integration.js

```diff
@@ -74,10 +74,11 @@ describe( 'dev-env/release-tools/utils', () => {
 			sandbox.restore();
 		} );
 
-		it( 'generates a changelog for the first time', () => {
+		// Done callback is called manually because these tests are slow and sometimes Mocha throws a timeout error.
+		it( 'generates a changelog for the first time', done => {
```

Comment:
These tests fail randomly. We can revert it and restart the build on CI when they fail if we don't want to fix it in this PR.